lib/screens/settings/settings_view_model.dart

```diff
@@ -98,53 +98,97 @@ abstract class SettingsViewModelBase with Store, ViewModel {
+                value: _isCrashManagerEnabled,
+                onChanged: _toggleCrashManager,
+              ),
+            ],
+          ),
+          // Throw Test Exception Button
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: DecoratedBox(
+                  decoration: BoxDecoration(
+                    borderRadius: BorderRadius.circular(10),
+                    border: Border.all(
+                        color: Theme.of(context).secondaryHeaderColor,
+                        width: 0.5),
+                  ),
+                  child: PlatformTextButton(
+                    onPressed: triggerException,
+                    child: Text(LocaleKeys.setting_screen_throw_test_exception
+                        .localized()),
+                  ),
+                ),
+              ),
+            ],
+          ),
         ],
       );
     } else {
       return const SizedBox.shrink();
     }
   }
 
-  List<ElevatedButton> getLangageListButton(BuildContext context) => [
-        getLangageButton(
-            LocaleKeys.langage_english_langage.localized(), 'en', context),
-        getLangageButton(
-            LocaleKeys.langage_french_langage.localized(), 'fr', context),
-        getLangageButton(
-            LocaleKeys.langage_spanish_langage.localized(), 'es', context)
+  void triggerException() => throw Exception();
+
+  List<Widget> getLangageListButton(BuildContext context) => [
+        _getLangageButton(
+            language: LocaleKeys.langage_english_langage.localized(),
+            languageCode: 'en',
+            context: context),
+        _getLangageButton(
+            language: LocaleKeys.langage_french_langage.localized(),
+            languageCode: 'fr',
+            context: context),
+        _getLangageButton(
+            language: LocaleKeys.langage_spanish_langage.localized(),
+            languageCode: 'es',
+            context: context)
       ];
 
-  ElevatedButton getLangageButton(
-          String langage, String langageCode, BuildContext context) =>
-      ElevatedButton(
-        onPressed: () async {
-          await context.setLocale(Locale(langageCode));
+  Widget _getLangageButton(
+      {required String language,
+      required String languageCode,
+      required BuildContext context}) {
+    if (context.locale.languageCode == languageCode) {
+      return const SizedBox.shrink();
+    }
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 10),
+      child: PlatformElevatedButton(
+        color: Theme.of(context).primaryColor,
+        padding: const EdgeInsets.all(10),
+        onPressed: () async => {
+          await context.setLocale(Locale(languageCode)),
           // ignore: use_build_context_synchronously
-          Phoenix.rebirth(context);
+          Phoenix.rebirth(context)
```

Comment:
I don't think we need `Phoenix.rebirth` because we only use it when we want to rebuild your application.
and when I click on a button to switch to another language I notice that the screen goes black for a second and then it shows up again and I think that it `Phoenix` that does this.